tests/plugins/links/test_urlize.py

```diff
@@ -102,10 +102,6 @@
         'my.long.domain.example.com',
         EXPECTED_PARAGRAPH_TEMPLATE % ('http://my.long.domain.example.com', 'my.long.domain.example.com')
     ),
-    (
-        'localhost',
-        '<p>localhosts</p>',
-    ),
```

Comment:
This test seemed fine, the error was a typo: *localhost* vs *localhost**s***. It's actually a regression test of your own change?